backends/p4fmt/options.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BACKENDS_P4FMT_OPTIONS_H_
+#define BACKENDS_P4FMT_OPTIONS_H_
+
+#include "frontends/common/options.h"
+#include "frontends/common/parser_options.h"
+
+namespace P4Fmt {
+
+class P4fmtOptions : public CompilerOptions {
+ public:
+    P4fmtOptions();
+    virtual ~P4fmtOptions() = default;
+    P4fmtOptions(const P4fmtOptions &) = default;
+    P4fmtOptions(P4fmtOptions &&) = delete;
+    P4fmtOptions &operator=(const P4fmtOptions &) = default;
+    P4fmtOptions &operator=(P4fmtOptions &&) = delete;
+
+    const cstring &outputFile() const;
+
+ private:
+    // File to output to.
```

Comment:
Use three slashes here. 
```suggestion
    /// File to output to.
```